app/uk/gov/hmrc/trackingconsentfrontend/assets/test/common/getTrackingConsentBaseUrl.spec.ts

```diff
@@ -0,0 +1,50 @@
+import getTrackingConsentBaseUrl from '../../src/common/getTrackingConsentBaseUrl'
+// @ts-ignore
+import fixture from '../fixtures/servicePageWithBannerMinimal.html'
+
+describe('getTrackingConsentBaseUrl', () => {
+  beforeEach(() => {
+    delete window.location
+    document.getElementsByTagName('html')[0].innerHTML = fixture
+  })
+
+  it('should return "" if running under https://www.tax.service.gov.uk', () => {
+    // @ts-ignore
+    window.location = {
+      host: 'www.tax.service.gov.uk'
+    }
+    const url = getTrackingConsentBaseUrl()
+
+    expect(url).toEqual('')
+  })
+
+  it('should return "http://localhost:12345" if running under http://localhost:9000/some-service', () => {
+    // @ts-ignore
+    window.location = {
+      host: 'localhost:12345'
```

Comment:
This doesn't match the test name (http://localhost:9000/some-service)

Reply:
Good catch, amended.